File: AccuPay/Domain/Government/PhilHealthBracket.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations

Namespace Global.PayrollSys

    <Table("payphilhealth")>
    Public Class PhilHealthBracket

        <Key>
        Public Property RowID As Integer?

        Public Property Created As Date

        Public Property CreatedBy As Integer?

        Public Property LastUpd As Date

        Public Property LastUpdBy As Integer?

        Public Property SalaryBracket As Integer

        Public Property SalaryRangeFrom As Decimal

        Public Property SalaryRangeTo As Decimal

        Public Property SalaryBase As Decimal

        Public Property TotalMonthlyPremium As Decimal

        Public Property EmployeeShare As Decimal

        Public Property EmployerShare As Decimal

        Public Property HiddenData As Boolean

    End Class

End Namespace

File: AccuPay.Core/Entities/PhilHealthBracket.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Core.Entities
{
    [Table("payphilhealth")]
    public class PhilHealthBracket
    {
        [Key]
        public int? RowID { get; set; }

        public DateTime Created { get; set; }

        public int? CreatedBy { get; set; }

        public DateTime LastUpd { get; set; }

        public int? LastUpdBy { get; set; }

        public int SalaryBracket { get; set; }

        public decimal SalaryRangeFrom { get; set; }

        public decimal SalaryRangeTo { get; set; }

        public decimal SalaryBase { get; set; }

        public decimal TotalMonthlyPremium { get; set; }

        public decimal EmployeeShare { get; set; }

        public decimal EmployerShare { get; set; }

        public bool HiddenData { get; set; }
    }
}